viewing_party/party.py

```diff
@@ -1,23 +1,180 @@
 # ------------- WAVE 1 --------------------
 
+from __future__ import unicode_literals
+
+
 def create_movie(title, genre, rating):
-    pass
+    if not title or not genre or not rating:
+        return None
+    movie = {
+        "title": title,
+        "genre": genre,
+        "rating": rating,
+    }
+    return movie
+def add_to_watched(user_data, movie):
+    updated_data = user_data.copy()
+    updated_data["watched"].append(movie)
+    return updated_data
+def add_to_watchlist(user_data, movie):
+    updated_data = user_data.copy()
+    updated_data["watchlist"].append(movie)
+    return updated_data
+def watch_movie(user_data, title):

```

Comment:
Great job completing your first project Ryan! For consistent spacing, add a line after each function so it looks like:
```
    return movie

def add_to_watched(user_data, movie):
    updated_data = user_data.copy()
    updated_data["watched"].append(movie)
    return updated_data

def add_to_watchlist(user_data, movie):
    updated_data = user_data.copy()
    updated_data["watchlist"].append(movie)
    return updated_data

def watch_movie(user_data, title):
```

You can see whitespace rules here in the style guide: https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements